vrx_ign/CMakeLists.txt

```diff
@@ -21,9 +22,10 @@ set(IGN_PLUGIN_VER ${ignition-plugin1_VERSION_MAJOR})
 find_package(ignition-rendering6 REQUIRED)
 set(IGN_RENDERING_VER ${ignition-rendering6_VERSION_MAJOR})
 find_package(sdformat12 REQUIRED)
+find_package(ignition-physics4 REQUIRED)
```

Comment:
This should be `physics5` as this is the version in Gazebo Fortress. See [this repository](https://github.com/gazebo-tooling/gazebodistro/blob/master/collection-fortress.yaml) for all the library versions in each Gazebo distribution. Change the next line too.